src/de/mod.rs

```diff
@@ -308,6 +311,34 @@ impl<'a, 'de> de::Deserializer<'de> for &'a mut Deserializer<'de> {
         deserialize_signed!(self, visitor, i64, visit_i64)
     }
 
+    serde_if_integer128! {
```

Comment:
Why do we need this macro? We only support 1.53+ so we can assume it is available, right?

Reply:
You are right. I missed that version assumption. I will remove it.